src/controller/age.controller.js

```diff
@@ -6,9 +6,10 @@ const limiter = require("../middleware/rateLimiter");
  * @returns {number}
  */
 
-router.get("/howold", limiter, async (req, res) => {
+router.get("/howold",limiter, async (req, res) => {
```

Comment:
Codacy has a fix for the issue: [Insert `·`](https://app.codacy.com/gh/armerillo/ql-task/pullRequest?prid=9976636)
```suggestion
router.get("/howold", limiter, async (req, res) => {
```